lib/rainforest/cli/http_client.rb

```diff
@@ -33,8 +33,8 @@ def post(url, body = {})
       JSON.parse(response.body)
     end
 
-    def get(url, body = {}, retries_on_failures: false)
-      wrap_exceptions(retries_on_failures) do
+    def get(url, body = {}, options = { retries_on_failures: false })
```

Comment:
It already was a hash to begin with. Just an implied hash <3 ruby

The original reason for it was because when an argument that is just True/False, it's not clear for the user of the method what the purpose is of that value is. At least to me.